drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImpl.java

```diff
@@ -147,8 +147,27 @@ public Path create( final Path resource,
                         final InputStream content,
                         final String sessionId,
                         final String comment ) {
-        final SessionInfo sessionInfo = getSessionInfo( sessionId );
-        log.info( "USER:" + sessionInfo.getIdentity().getIdentifier() + " CREATING asset [" + resource.getFileName() + "]" );
+        return create(resource, content, sessionId, comment, true);
+    }
+
+    @Override
+    public Path save( final Path resource,
+                      final InputStream content,
+                      final String sessionId,
+                      final String comment ) {
+        return create(resource, content, sessionId, comment, false);
+    }
+
+    Path create( final Path resource,
```

Comment:
Personally, I'd prefer this to be called ```writeToFile(..)``` rather than ```create(..)``` since it doesn't create for the "update scenario".. also it'd be good to have the code formatter ran over this; as the indentation is screwy.